utils/mf5to6/src/MF2005/pcgn2.f90

```diff
@@ -83,7 +83,7 @@ SUBROUTINE PCGN2AR(IN,FREE_FMT,ITER_MO,IGRID)
           STOP
        ELSE
           IF (LEN_TRIM(CHAR_STRING)==0) CYCLE
-          CHECK=CHAR_STRING
+          CHECK=CHAR_STRING(1:1)
```

Comment:
This src file is directly from MODFLOW-2005. I doubt we will ever update it with the original source file from MODFLOW-2005 but this change would be lost if we did. Should we make the same change in MODFLOW-2005 and MODFLOW-NWT??

Reply:
MODFLOW-2005 (1.2.0) has six character truncation issues, including this one (the others are in other files that don't appear to be part of mf5to6). I can provide a patch for this later. MODFLOW-NWT doesn't have any character truncation issues, and this line does not appear in it.

Reply:
xref https://github.com/MODFLOW-USGS/mf2005/pull/45